### DISCUSSION TITLE
Does anyone need this kind of thing? *(Win11 ARM)

### DISCUSSION THREAD

**Discussion by biennvops** [WhatTheBlock/WindowsSimplify#83] [General]
Yes - as long as I can add other keyboard layouts (Japanese to be exact) :) otherwise, well...

**Comment by WhatTheBlock** [comment]
Sorry, that version cannot install language packs. I have prepared up to 4 versions this time, and the version with the most complete functions can work normally.

![Snipaste_2023-07-12_19-17-01-min](https://github.com/WhatTheBlock/WindowsSimplify/assets/34786954/cfc4eb3f-d47b-4dc0-94dc-e162596f8f13)

Update:
Ver2 and others doesn't support.

![Snipaste_2023-07-12_19-25-01-min](https://github.com/WhatTheBlock/WindowsSimplify/assets/34786954/013def11-17da-4713-aab9-2e2456b5e027)

**Comment by biennvops** [comment]
@WhatTheBlock What about copying the IME folder as a workaround you said before? Does it work?

**Reply by WhatTheBlock** [reply]
Ok I'll test it out.

**Comment by WhatTheBlock** [comment]
@BienGudBoy 
I made a new level 2, which can install language packs and input methods normally. But I also used the workaround mentioned before. Not sure if other languages are supported.

![Snipaste_2023-07-13_08-54-23-min](https://github.com/WhatTheBlock/WindowsSimplify/assets/34786954/54420967-e002-4a8b-b24b-cce58dbcb5b0)

**Reply by biennvops** [reply]
Can confirm that I can install language packs just fine on the new V2 build.
However, the size is quite large compared to the old ones. What about copying the IME folder to V3/V5?

**Reply by WhatTheBlock** [reply]
> However, the size is quite large compared to the old ones.

This is because I copied the Japanese IME dictionary into the system.

> What about copying the IME folder to V3/V5?

The deleted components of these two versions will cause Windows Update to no longer work. Therefore any additional components cannot be added, such as language packs.

**Comment by biennvops** [comment]
> The deleted components of these two versions will cause Windows Update to no longer work. Therefore any additional components cannot be added, such as language packs.

What about adding IME first, then remove the components? Will it work?
